crates/bench/src/spacetime_module.rs

```diff
@@ -0,0 +1,232 @@
+use spacetimedb::db::{Config, FsyncPolicy, Storage};
+use spacetimedb_lib::{sats::ArrayValue, AlgebraicValue, ProductValue};
+use spacetimedb_testing::modules::{start_runtime, CompiledModule, ModuleHandle};
+use tokio::runtime::Runtime;
+
+use crate::{
+    database::BenchDatabase,
+    schemas::{snake_case_table_name, table_name, BenchTable},
+    ResultBench,
+};
+
+lazy_static::lazy_static! {
+    pub static ref BENCHMARKS_MODULE: CompiledModule =
+        CompiledModule::compile("benchmarks");
+}
+
+pub struct SpacetimeModule {
+    runtime: Runtime,
+    // it's necessary for this to be dropped BEFORE the runtime.
```

Comment:
I personally prefer this (an explicit Drop impl) to e.g. relying on a comment about relying on the drop ordering based on field order. Maybe Mazdak or Noa have a stronger style opinion about it, I'd be curious.